cloud/gcp/storage/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "region" {
+  type        = string
+  description = "Default region for bucket resources"
+}
+
+variable "company-name" {
```

Comment:
suggestion (blocking): recommend renaming to "organization-name" since this code will be deployed by civic entities (e.g. city governments) rather than companies

Reply:
woops, looks like it should be `application_name`: https://github.com/seattle-uat/civiform/blob/main/cloud/azure/variables.tf#L1

Reply:
Done